app/javascript/widget/components/HeaderActions.vue

```diff
@@ -42,6 +71,15 @@ export default {
     showHeaderActions() {
       return this.isIframe || this.isRNWebView;
     },
+    isResolved() {
+      if (this.conversationAttributes.status) {
```

Comment:
Do we need to check this? It would available always right?

Reply:
Updated the logic.